### ISSUE TITLE
https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-connect-wordpress-to-mysql-managed-database

### ISSUE THREAD

**Issue by yingdi11** [awsdocs/amazon-lightsail-developer-guide#17]
https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-connect-wordpress-to-mysql-managed-database

mysqldump -u root --databases bitnami_wordpress --single-transaction --compress --order-by-primary -p$(cat /home/bitnami/bitnami_application_password) | mysql -u DbUserName --host DbEndpoint --password

will result in below errors 

ERROR 1045 (28000): Access denied for user 'dbmasteruser'@'172.26.3.110' (using password: 
NO)

**Comment by rlatino** [comment]
Hi @yingdi11 ,

I am looking into this. I'll follow up as soon as I have more info. If I find a solution, I'll update the doc and let you know after it's published.

**Comment by maximedupre** [comment]
@rlatino  Any update?

**Comment by rlatino** [comment]
Hi @maximedupre ,

I reviewed the doc, and made a couple of edits that should resolve the issue. I tested the procedure with the edits and it works. Give it a try and let me know what you think. Closing this issue unless you have further difficulties with the procedure.

View the diff: [https://github.com/awsdocs/amazon-lightsail-developer-guide/commit/de5bcde3b9a379a74b5875568a27b60c913aa2a1?short_path=f34a9c1#diff-f34a9c194df3874e3f823c0337f53ca3206407982733d33167ff40ff0da2b37a](https://github.com/awsdocs/amazon-lightsail-developer-guide/commit/de5bcde3b9a379a74b5875568a27b60c913aa2a1?short_path=f34a9c1#diff-f34a9c194df3874e3f823c0337f53ca3206407982733d33167ff40ff0da2b37a)

View the updated doc: https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-connect-wordpress-to-mysql-managed-database
